ops/storage-ddls/cassandra-setup.cql

```diff
@@ -175,27 +175,27 @@ CREATE TABLE popularplaces (
 
 DROP TABLE IF EXISTS eventtopics;
 CREATE TABLE eventtopics(
-    eventids set<text>,
+    eventid text,
     topic text,
-    insertiontime timestamp,
+    eventtime timestamp,
     pipelinekey text,
     externalsourceid text,
-    PRIMARY KEY ((topic), pipelinekey, externalsourceid)
+    PRIMARY KEY ((topic), pipelinekey, externalsourceid, eventtime, eventid)
 );
 
 DROP TABLE IF EXISTS eventplaces;
 CREATE TABLE eventplaces(
-    eventids set<text>,
+    eventid text,
     conjunctiontopic1 text,
     conjunctiontopic2 text,
     conjunctiontopic3 text,
     centroidlat double,
     centroidlon double,
     placeid text,
-    insertiontime timestamp,
+    eventtime timestamp,
     pipelinekey text,
     externalsourceid text,
-    PRIMARY KEY ((placeid), conjunctiontopic1, conjunctiontopic2, conjunctiontopic3, pipelinekey, externalsourceid, centroidlat, centroidlon)
+    PRIMARY KEY ((placeid, conjunctiontopic1, conjunctiontopic2, conjunctiontopic3), pipelinekey, externalsourceid, eventtime, eventid)
```

Comment:
Also note that querying on placeid is more precise than centroid locations since there can be multiple places for the same location but every id is guaranteed to refer to only one location.